src/users/containers/user-details/user-details.component.ts

```diff
@@ -67,6 +74,26 @@ export class UserDetailsComponent implements OnInit, OnDestroy {
 
     this.user$ = this.userStore.pipe(select(fromStore.getUserDetails));
 
+    combineLatest([organisationAccessTypes$, this.user$, getOgdInviteUserFlowFeatureIsEnabled$])
+      .pipe(takeUntil(this.onDestory$))
+      .subscribe(([organisationAccessTypes, user, isFeatureEnabled]) => {
+        user = { ...user, accessTypes: JSON.parse(userAccessTypesExample) as UserAccessType };
```

Comment:
this line can be removed now that the node api layer now injects an access type instead of having to add it in multiple components